packages/rum/src/boot/rum.entry.spec.ts

```diff
@@ -1,11 +1,15 @@
-import { ErrorSource, ONE_SECOND } from '@datadog/browser-core'
+import { Configuration, ErrorSource, ONE_SECOND } from '@datadog/browser-core'
 import { setup, TestSetupBuilder } from '../../test/specHelper'
 import { ActionType } from '../domain/rumEventsCollection/action/trackActions'
 import { makeRumGlobal, RumGlobal, RumUserConfiguration, StartRum } from './rum.entry'
 
 const noopStartRum = () => ({
   addAction: () => undefined,
   addError: () => undefined,
+  addTiming: () => undefined,
+  configuration: ({
+    isEnabled: () => true,
+  } as any) as Configuration,
```

Comment:
is it needed?

Reply:
Yes, otherwise Typescript will complain about it.

Reply:
My bad not clear enough, I was more concerned about the `true` in `isEnabled: () => true`

Reply:
The return of `isEnabled` should be `true` for `addTiming` to be added to `RumGlobal`. Do you think we should make it return `false` by default and only mock it to return `true` when we actually need to test `addTiming`?

Reply:
Not critical here, but since the general case would be not enabled, I'd be in favor of that yes